pkg/brokerapi/service_instance.go

```diff
@@ -40,6 +40,20 @@ type CreateServiceInstanceRequest struct {
 	SpaceID           string                 `json:"space_guid,omitempty"`
 	Parameters        map[string]interface{} `json:"parameters,omitempty"`
 	AcceptsIncomplete bool                   `json:"accepts_incomplete,omitempty"`
+	ContextProfile    ContextProfile         `json:"context,omitempty"`
+}
+
+// ContextProfilePlatformKubernetes is a constant to send when the
+// client is representing a kubernetes style ecosystem.
+const ContextProfilePlatformKubernetes string = "kubernetes"
+
+// ContextProfile implements the optional OSB field
+// https://github.com/duglin/servicebroker/blob/CFisms/context-profiles.md#kubernetes
+type ContextProfile struct {
+	// Platform is always `kubernetes`
```

Comment:
Not sure it matters much since I would hope the broker would know that based on the auth info - each cluster should be unique in that regard.  But, if you know how to get that info I'm not against adding it - now or later.